src/languages/en.js

```diff
@@ -542,6 +542,9 @@ export default {
             firstNameLength: 'First name shouldn\'t be longer than 50 characters',
             lastNameLength: 'Last name shouldn\'t be longer than 50 characters',
             characterLimit: ({limit}) => `Exceeds the max length of ${limit} characters`,
+            hasCommas: ({invalidCharacter}) => `Please remove the ${invalidCharacter} from the name field.`,
```

Comment:
We should rename this
```suggestion
            hasInvalidCharacter: ({invalidCharacter}) => `Please remove the ${invalidCharacter} from the name field.`,
```